### PR TITLE
feat(w3c/groups): expose WG patentPolicy info 

### PR DESCRIPTION
**Pull request by sidvishnoi**
Closes https://github.com/marcoscaceres/respec.org/issues/108

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
Maybe `null` in that one place...

File: routes/w3c/group.js [hunk 1]
```diff
@@ -85,16 +86,39 @@ async function fetchGroupInfo(id, shortname, type) {
   const json = await res.json();
 
   const { name, _links: links } = json;
+
+  const patentPolicy = links["active-charter"]
+    ? await getPatentPolicy(links["active-charter"].href)
+    : undefined;
```
[the remark this answers is not in the document]
Comment (sidvishnoi) [reply routes/w3c/group.js:92]: `undefined` gets removed from JSON response, which maybe makes sense here (like in CGs).
https://github.com/marcoscaceres/respec.org/pull/111/files#diff-b53ff28c2dd59444a04d583fba7fe4e5R114 should probably return `null`. WDYT?

[the remark this answers is not in the document]
Comment (sidvishnoi) [reply routes/w3c/group.js:92]: URI and patentURI can also be undefined at present. Should we switch them to null also?

[the remark this answers is not in the document]
Comment (marcoscaceres) [reply routes/w3c/group.js:92]: I guess removing them makes sense... depends if the other side is expecting them to be there... and can be done away with `null`... so in some way it's nice to have null... you give back what was asked for, but `null` explicitly says it wasn't available, not `undefined`.

[the remark this answers is not in the document]
Comment (sidvishnoi) [reply routes/w3c/group.js:92]: Lets return `null` when we fail to get patentPolicy (`PP2017 | PP2020 | null`), and keep undefined in API response for consistency (like CGs don't have active-charter).

**Pull request merged**